lib/reverse_sentence.rb

```diff
@@ -1,4 +1,61 @@
 # A method to reverse the words in a sentence, in place.
+require 'pry'
 def reverse_sentence(my_sentence)
-  raise NotImplementedError
+  return if my_sentence == nil || my_sentence.length < 1
+  i = 0
+  j = my_sentence.length - 1
+  until i == j
+    # if my_sentence[j] == nil
+    #   my_sentence[j] = " "
+    # elsif my_sentence[i] == nil
+    #   my_sentence[i] = " "
+    # end
+    temp = my_sentence[i]
+    my_sentence[i] = my_sentence[j]
+    my_sentence[j] = temp
+    i += 1
+    j -= 1
+  end
+  return reverse_words(my_sentence)
```

Comment:
Since you're manipulating the string in-place, you don't need to return anything.